src/claim/authorize-ksign-secp256k1/authorize-ksign-secp256k1.test.js

```diff
@@ -0,0 +1,62 @@
+const chai = require('chai');
+const ethUtil = require('ethereumjs-util');
+const Claim = require('../claim');
+const authorizeKSignSecp256k1 = require('./authorize-ksign-secp256k1');
+const utils = require('../../utils');
+const CONSTANTS = require('../../constants');
+
+const { secp256k1 } = ethUtil;
+const { expect } = chai;
+
+describe('[Claim Authorize KSignSecp256k1]', () => {
```

Comment:
I don't see any test to check that the code is treating well any not right value... I mean, i.e.:
What does happen when try to parse and don't send the right structure or has not version?